prow/deck/deployment.yaml

```diff
@@ -42,6 +42,19 @@ spec:
             - name: extensions
               mountPath: /static/extensions
               readOnly: true
+      livenessProbe:
+        httpGet:
+          path: /healthz
+          port: 8081
+        initialDelaySeconds: 3
+        periodSeconds: 3
+      readinessProbe:
+        httpGet:
+          path: /healthz/ready
+          port: 8081
+        initialDelaySeconds: 10
+        periodSeconds: 3
+        timeoutSeconds: 600
```

Comment:
`timeoutSeconds` maybe seems a bit long, we would attempt 20 connections before timing out. I'd be tempted to set the timeout much closer to the period

Reply:
@JoelSpeed this is readiness probe though, mostly for first-time deploying no? I reckon those can take a while?

Reply:
Though I'm ok with setting lower

Reply:
I'd be quite concerned if it took 10 minutes for the program to respond to a ping though 😆 This timeout is how long K8s will keep the connection open before assuming this attempt failed. So if we set the timeout to 3 seconds then we should always close the first attempt before starting the next attempt, which I think makes sense to do? WDYT?